clusters/cijioagents2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+helmDefaults:
+  atomic: true
+  force: false
+  timeout: 300
+  wait: true
+repositories:
+  # https://github.com/DataDog/helm-charts/
+  - name: datadog
+    url: https://helm.datadoghq.com
+  # https://github.com/jenkins-infra/helm-charts/
+  - name: jenkins-infra
+    url: https://jenkins-infra.github.io/helm-charts
+releases:
+  - name: docker-registry-secrets
+    # This helm chart doesn't create any resources within the namespace specified below.
+    # Specifying a namespace is required by the "needs" feature of helmfile (to allow referencing to this release from others)
+    namespace: default
+    chart: jenkins-infra/docker-registry-secrets
+    version: 0.1.0
+    values:
+      - "../config/docker-registry-secrets.yaml"
+    secrets:
+      - "../secrets/config/docker-registry-secrets/secrets.yaml"
+  - name: datadog
+    needs:
+      - default/docker-registry-secrets
+    namespace: datadog
+    chart: datadog/datadog
+    version: 3.83.0
+    values:
+      - "../config/datadog.yaml.gotmpl"
+      - "../config/datadog_cijioagents2.yaml"
+    secrets:
+      - "../secrets/config/datadog/cijioagents2-secrets.yaml"
+  - name: jenkins-agents
+    needs:
+      - default/docker-registry-secrets
+    namespace: jenkins-agents
+    chart: jenkins-infra/jenkins-kubernetes-agents
+    version: 1.0.0
+    values:
+      - "../config/jenkins-kubernetes-agents_ci.jenkins.io_cijioagents2.yaml"
+    secrets:
+      - "../secrets/config/jenkins-kubernetes-agents/secrets.yaml"
+  - name: jenkins-agents-bom
+    namespace: jenkins-agents-bom
+    chart: jenkins-infra/jenkins-kubernetes-agents
+    version: 1.0.0
+    values:
+      - "../config/jenkins-kubernetes-agents_ci.jenkins.io_cijioagents2-bom.yaml"
+    secrets:
+      - "../secrets/config/jenkins-kubernetes-agents/secrets.yaml"
+  - name: artifact-caching-proxy
+    namespace: artifact-caching-proxy
+    chart: jenkins-infra/artifact-caching-proxy
+    version: 1.6.2
+    values:
+      - "../config/artifact-caching-proxy_azure-aks.yaml"
```

Comment:
- Is there a sized node pool already to support this release?
- The config file here is clearly invalid: it has `aks` in its name ;)